docs/DeveloperGuide.adoc

```diff
@@ -279,104 +298,446 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
-* is reasonably comfortable using CLI apps
+* money-conscious NUS students
+* number of transactions are significant and needs to be managed
+* apps are preferred on desktop over other platforms
+* typists that type quickly and prefers typing over mouse input
+* CLI usage does not pose any discomfort
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*:
+
+* provides a more effective financial management compared to typical mouse/GUI driven app
+* plans for future spending are made easier
+* congregates all info regarding fees and costs in a single location
 
 [appendix]
 == User Stories
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="100%",cols="5%,<25%,<30%,<40%",options="header",]
```

Comment:
Some of the user stories seem similar and are excessively specific on a per-user basis, without specifying any extra features expected of the application. For example, user story numbers 10 and 17 both relate to viewing a common expense tracker report, and their benefit seems a bit contrived.